### ISSUE TITLE
Australian Data redux

### ISSUE THREAD

**Issue by Lingtax** [ropensci/ozunconf18#6]
At the 2017 unconference, some of us worked on [making Australian data more accessible](https://github.com/ropensci/ozunconf17/issues/17). This led to [ozflights](https://github.com/ropenscilabs/ozflights) and [ozroaddeaths](https://github.com/ropenscilabs/ozroaddeaths).

Maybe we can do more of this this year? What data are you interested in / do you think might be useful?

I found this [data on Liquor and Gambling in Victoria](https://www.vcglr.vic.gov.au/resources/data-and-research) maybe we could find things like this nationally and stitch it together?

**Comment by njtierney** [comment]
Absolutely! This sounds great! #4 discusses an idea for an Australian babynames pkg (or rather, regular names). 

Some ideas:

* ozflights and ozroaddeaths 
    * Update with current data
    * Explore options for improving automation for updating them
    * expand on / improve documentation / vignettes
* ozgambling + ozliqour
    * create an API pkg or clean up and combine current data you linked
    * potentially try and link with other data (ozroaddeaths? I'm not sure if the resolution of the data are compatible, though)

* Explore the [city of Melbourne data](https://data.melbourne.vic.gov.au/) - or other cities around Australia?

* Explore the [`datagovau` package](https://github.com/ropenscilabs/datagovau)

* Provide a central listing of existing data packages for Australia

**Comment by njtierney** [comment]
Ah, awesome! Thanks @peggynewman !

**Comment by jesse-jesse** [comment]
I think this is a great idea. I was thinking about conducting an audit of the available open data. To report on how open it is practically. I recently requested some data from QLD and they provided it in a PDF. Supposedly that is open data

**Comment by Lingtax** [comment]
I think @djnavarro [agrees with you](https://twitter.com/djnavarro/status/1065039731690418177)

**Comment by jesse-jesse** [comment]
I started looking at the QLD gambling data a while ago. But we didn't get too far. https://github.com/RedNigel/Queensland-gaming-machines . 

Could combine with the vic gambling data.

**Comment by Lingtax** [comment]
The accessibility score idea is gold, @jesse-jesse 

Are there standard metrics we could use, or would we need to derive these? If so, I'd be keen to lock down our criteria early and register them in some timestamped way to protect against arguments of cherrypicking/target shifting.

**Comment by jesse-jesse** [comment]
I haven't looked for any metrics yet. I am sure we could find some. Good idea to lock them down. I think they should be able to be re-evaluated, but the re-evaluation should be transparent. the unconf could also be a good place to vet the accessibility score as well. we could do a first draft in the morning and then review it at lunch or morning tea and get the input of others.

**Comment by Lingtax** [comment]
Ohhh... I got one : https://www.ands-nectar-rds.org.au/fair-tool 
Worth considering as a starting point, as it links to some actionable goals and there are efforts to promote these criteria; https://ardc.edu.au/planning/events/top-10-fair-data-things-global-sprint

**Comment by mdsumner** [comment]
Seems like https://nationalmap.gov.au/about.html by terria.io is the latest and greatest central source, it's mostly new to me
